clang/lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -1354,7 +1354,9 @@ unsigned UnwrappedLineFormatter::format(
 
     bool FixIndentation = (FixBadIndentation || ContinueFormatting) &&
                           Indent != TheLine.First->OriginalColumn;
-    bool ShouldFormat = TheLine.Affected || FixIndentation;
+    bool ShouldFormat = (TheLine.Affected || FixIndentation);
```

Comment:
Why adding the parenthesis? (Actually clang-format, if new enough, should remove them.)

Reply:
this was unintentional, fixed now